lib/testutils/src/git.rs

```diff
@@ -56,23 +56,26 @@ pub fn init_bare(directory: impl AsRef<Path>) -> gix::Repository {
     .to_thread_local()
 }
 
-pub fn clone(dest_path: &Path, url: &str) -> gix::Repository {
-    let mut prepare_fetch = gix::clone::PrepareFetch::new(
-        url,
-        dest_path,
-        gix::create::Kind::WithWorktree,
-        gix::create::Options::default(),
-        open_options(),
-    )
-    .unwrap();
-    let (mut prepare_checkout, _outcome) = prepare_fetch
-        .fetch_then_checkout(gix::progress::Discard, &gix::interrupt::IS_INTERRUPTED)
-        .unwrap();
-    let (repo, _outcome) = prepare_checkout
-        .main_worktree(gix::progress::Discard, &gix::interrupt::IS_INTERRUPTED)
+pub fn clone(dest_path: &Path, repo_url: &str) -> gix::Repository {
+    // gitoxide doesn't seem to write the `refs/remotes` to disk on clone
+    // (https://github.com/GitoxideLabs/gitoxide/issues/1865)
+    //
+    // it'd be nice not to have the hard-dependency on `git` at test time, but it
+    // seems unavoidable
+    let output = std::process::Command::new("git")
+        .args(["clone", repo_url])
+        .arg(dest_path)
+        .output()
         .unwrap();
-
-    repo
+    assert!(
+        output.status.success(),
+        "git cloning failed with exit code {:?}:\n{}\n----- stderr -----\n{}",
+        output.status.code(),
```

Comment:
nit: `output.status` implements `Display`, so this could be `"git cloning failed with {} ...", output.status, ...`